code/game/gamemodes/cult/cult_structures.dm

```diff
@@ -110,6 +110,10 @@
 	start_process()
 
 
+/obj/structure/cult/pylon/Destroy()
+	processing_objects.Remove(src)

```

Comment:
theoretically correct. Though it is best practice to still nuke them, primarily to avoid circular references and to be explicit about it.

/ASCIIshrug.